api/app/Models/User.php

```diff
@@ -348,6 +348,40 @@ public function filterByClassifications(Builder $query, array $classifications):
             });
 
             $this->orFilterByClassificationToSalary($query, $classifications);
+           $this->orfilterByClassificationToGenericJobTitles($query, $classifications);
```

Comment:
Indentation inconsistency
```suggestion
            $this->orfilterByClassificationToGenericJobTitles($query, $classifications);
```